File: CS/Form1.cs
using System;
using System.Windows.Forms;
// ...

namespace PreviewSort {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            XtraReport1 report = new XtraReport1();
            report.SetReportParameter(11077);
            report.ShowPreview();
        }

    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms
' ...

Namespace PreviewSort
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim report As New XtraReport1()
			report.SetReportParameter(11077)
			report.ShowPreview()
		End Sub

	End Class
End Namespace